Automattic/newspack-plugin
# Engagement: remove MJML creds settings

### All Submissions:






### Changes proposed in this Pull Request:



With https://github.com/Automattic/newspack-newsletters/pull/443, MJML API keys are no longer required.

### How to test the changes in this Pull Request:

1. Visit the Setup Wizard, "Services" step, observe there are no MJML settings inputs
2. Visit the Engagement Wizard, observe the same thing

### Other information:
